src/stores/widgets/ElementWidgetActions.ts

```diff
@@ -19,6 +19,7 @@ import { IWidgetApiRequest } from "matrix-widget-api";
 export enum ElementWidgetActions {
     ClientReady = "im.vector.ready",
     HangupCall = "im.vector.hangup",
+    StartLiveStream = "im.vector.start_live_stream",
```

Comment:
oh, we should probably start using `io.element` instead

Reply:
Personally I'm still an advocate of just sticking with vector / im.vector as an internal codename for everything in the project, independent of the branding.

Reply:
yea, I suppose we would need to fix all the other vector instances if we wanted to transition properly.